python/packages/nisar/products/writers/BaseWriterSingleInput.py

```diff
@@ -664,10 +664,9 @@ def populate_identification_common(self):
             processing_type = np.bytes_('Nominal')
         elif processing_type_runconfig == 'UR':
             processing_type = np.bytes_('Urgent')
-        elif processing_type_runconfig == 'OD':
-            processing_type = np.bytes_('Custom')
         else:
-            processing_type = np.bytes_('Undefined')
+            processing_type = np.bytes_('Custom')
```

Comment:
In the corresponding InSAR PR, @vbrancat added a log message to the `else` conditional.
* Discussion: https://github.com/isce-framework/isce3/pull/60#discussion_r2219932597
* Implementation: https://github.com/isce-framework/isce3/pull/60/commits/7a3b8c5d476f4b821ad6e0fe613a183f17c37353

Could you please update GSLC/GCOV writer to match?

Reply:
Thank you, @nemo794 , the warning message was added in the commit https://github.com/isce-framework/isce3/pull/59/commits/49174626aca111f6df1fb9b8a4e51e45226fd65c.

Reply:
This is great! Thank you for logging the behavior. Very helpful for future developers.

...Except that ["OD" is also a valid value in the runconfig](https://github-fn.jpl.nasa.gov/isce-3/isce/blob/b38758ba6abb80c17726941e775d0657a0599ad4/share/nisar/schemas/gcov.yaml#L59), but this log message as-is would emit a warning that "OD" is invalid.

Could you please accommodate that?

Reply:
Whoops. Yep, this was updated in the commit https://github.com/isce-framework/isce3/pull/59/commits/044d37fdd064829347b032ca8a47148bbfa48f90. Thanks.